Python based mysql db installer script

rcythr opened the pull request:
Reimplements the mysql setup script. This version is more maintainable, cross platform, and has more options.

rcythr committed:
Cleaned up the sql batch installer

Cleaned up the sql batch installer so that it makes use of 'dir' a little more wisely. Also fixed the issues with re-installing of some databases multiple times.

rcythr committed:
Reimplements the mysql setup script. This version is more maintainable, cross platform, and has more options.

apathyboy commented:
Having an issue where it doesn't appear to be calling the create.sql script before the other scripts, this causes the other scripts to get angry that their db doesn't exist

apathyboy commented:
I see the create script is getting called first in my logs, the problem I think is that it's getting passed to callMysql which adds a --database field. Since the database doesn't exist yet the create script fails and so do all the subsequent scripts.

rcythr commented:
That's a good catch. I'll make adjustments to get that fixed up.

rcythr committed:
Fixed issues with create.sql

Fixed with the sql script that sets up the db which made mysql mad and caused issues with installation on brand new installs.

rcythr committed:
Minor syntax changes to prevent more mysql anger.

Not sure if this is necessary, but doing it anyway to make sure.

rcythr committed:
Finally got home to test this. Fixed another minor issue. Script now works as expected for a fresh db.

rcythr commented:
Okay. It's all fixed up now.

apathyboy commented:
Great, works like a charm now.
